plover_auto_reconnect_machine.py

```diff
@@ -20,11 +20,12 @@ def __init__(self, engine: StenoEngine):
         self._engine: StenoEngine = engine
         self._lock = Condition()
         self._stop = Event()
+        self._thread = Thread(self)
```

Comment:
```suggestion
        self._thread = Thread(target=self.run)
```

With this change, the reconnection part still work. For the other issue, it seems obvious that it wouldn't happen, but I don't really have any deterministic way to test it.

Reply:
Currently I work inside a virtual Linux machine on a Windows Host. I configured VMWare to connect the serial port of my ErgoDox to the virtual machine. Inside the virtual machine the port is always connected and plover never looses the connection. The only downside is, that I need to manually reconnect the port to the virtual machine by clicking on a button in VMWare.

Before that, I used https://github.com/tschulte/ComPortPipe, but on my new Windows host com0com did not work any more and I started using the above.